src/datatable.js

```diff
@@ -195,7 +191,7 @@ export class DataTable {
       labels.push({
         nested: cleanedColumn.indexOf('.') !== -1,
         column: cleanedColumn,
-        label : ucfirst(clean(aliased[1] || aliased[0]))
+        label : clean(aliased[1] || aliased[0])
```

Comment:
So now you're required to alias all your columns.

Why do you need this? Because I want to close this PR.